### DISCUSSION TITLE
Include Nuxt3 app with all its functions and routes in another app.

### DISCUSSION THREAD

**Discussion by natear22** [nuxt/nuxt#20569] [Questions] [answered]
Hi!

I have built a git-based headless cms with nuxt 3 and I want to be able to add it to other Nuxt 3 apps.
How can I do it? How can I include the first app with all its routes into another app ?

**Comment by warflash** [comment, marked as answer]
[Nuxt layers](https://nuxt.com/docs/getting-started/layers) should be what you're looking for, you could also create a [module](https://nuxt.com/docs/guide/going-further/modules/#module-author-guide)

**Reply by natear22** [reply]
Thank you!
